docs/subnets/create-a-vm-blobvm.md

```diff
@@ -0,0 +1,1359 @@
+# How to Build a Complex VM
+
+This is part of a series of tutorials for building a Virtual Machine (VM):
+
+- [Introduction to Virtual Machines](./introduction-to-vm.md)
+- [How to Build a Simple VM](./create-a-vm-timestampvm.md)
+- How to Build a Complex VM (this article)
+
+## Introduction
+
+In this tutorial, we will learn all about how we can build a virtual machine by referencing [BlobVM](https://github.com/ava-labs/blobvm). It's a Virtual Machine that can be used to instantiate key-value blockchains for storing files like images, videos, etc. in an efficient way.
```

Comment:
I don't think it's true that BlobVM will scale well for images/videos, so we should probably remove those references.

Reply:
I removed this reference and just said generic k/v storage.

Totally an aside but I think it kind of does scale fine but it would depend on the use-case/if you consider gas prices an issue in scaling :p